test/conftest.py

```diff
@@ -144,8 +144,8 @@ def opt_ml():
 
 @pytest.fixture(autouse=True)
 def skip_wrong_toolkit(request, toolkit):
-    run_coach = request.node.get_marker('run_coach')
-    run_ray = request.node.get_marker('run_ray')
+    run_coach = request.node.get_closest_marker('run_coach')
+    run_ray = request.node.get_closest_marker('run_ray')
```

Comment:
I'm confused haha: https://github.com/aws/sagemaker-rl-container/blob/master/test/conftest.py#L147-L148. This PR is against master?